pkg/storage/run.go

```diff
@@ -45,7 +46,7 @@ type Run struct {
 
 	// Bundle is the definition of the bundle.
 	// Bundle has custom marshal logic in MarshalJson.
-	Bundle bundle.Bundle `json:"-"`
+	Bundle bundle.Bundle `json:"-" gorm:"json"`
```

Comment:
I have not used Gorm a lot, but that this mean that the custom logic in `MarshalJson` will be executed for when stored?

Reply:
Yes, types can implement the `json.Marshaller` interface to change their default json marshal structure. It will store that output in the db.